sites/default/images/logos/core/login/primary/README

```diff
@@ -0,0 +1,11 @@
+# Primary Login Logo
```

Comment:
just to confirm that this mechanism will not break anything if this directory does not exist (for example, upgraders will not have this directory).